packages/react/.storybook/story-config.ts

```diff
@@ -56,8 +56,11 @@ const StoryConfig: StorybookConfig = {
         },
       }
     },
-    hierarchy: `${StorybookCategories.Surfaces}/AppBar`,
+    hierarchy: `${StorybookCategories.Surfaces}/TopNav`,
```

Comment:
Should this be `TopNav` or `Top Navigation`? WDYT?

Reply:
Changed to `Top Navigation`